contrib/docker/start.sh

```diff
@@ -12,7 +12,7 @@ php artisan migrate --force
 php artisan update
 
 # Run a worker if it is set as embedded
-if [ $HORIZON_EMBED = true ]; then
+if [ "$HORIZON_EMBED" -eq "true" ]; then
```

Comment:
-eq is for integers, = is well the comparison operator to use for strings.

The real bug was the lack of "" (and that should be fixed).